pkg/api/types/image_types.go

```diff
@@ -191,12 +193,13 @@ type ImagePullOptions struct {
 	Stderr        io.Writer
 	GOptions      GlobalCommandOptions
 	VerifyOptions ImageVerifyOptions
-	// Unpack the image for the current single platform (auto/true/false)
-	Unpack string
-	// Pull content for a specific platform
-	Platform []string
-	// Pull content for all platforms
-	AllPlatforms bool
+	// Unpack the image for the current single platform.
+	// If nil, it will unpack automatically if only 1 platform is specified.
+	Unpack *bool
+	// Content for specific platforms
+	OCISpecPlatform []v1.Platform
```

Comment:
Given we're removing the `AllPlatforms` boolean, can we add a comment here saying "empty for all platform"?